spring-boot-servlet/src/main/java/com/baeldung/props/PropertyLoader.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.props;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertyLoader {
+    private static final Logger log = LoggerFactory.getLogger(PropertyLoader.class);
+
+    public Properties getProperties(String file) {
+        Properties prop = new Properties();
+        InputStream input = null;
+        try {
+          input = getClass().getResourceAsStream(file);
+          prop.load(input);
+        } catch (IOException ex) {
+          if (input != null) {
+            input.close();
+          }
+        } finally {
```

Comment:
In Eclipse, I see a compiler warning (finally block does not complete normally).
May be a moot point if you no longer need this class.